data_hub_api/docmaps/v2/sql/docmaps_index.sql

```diff
@@ -330,13 +343,17 @@ t_result_with_sorted_manuscript_versions_array AS (
         result.editor_details,
         result.senior_editor_details,
         result.author_names_csv,
-        result.evaluations
+        result.evaluations,
+        CONCAT(publication.publication_date, ' ', publication.utc_publication_time) AS rp_publication_timestamp
```

Comment:
At this point it seems to be a datetime string as it is not parsed. It also doesn't include the timezone (which we assume to be UTC).